Automattic/Gravatar-SDK-iOS
# Delete avatar

Closes #544 

### Description

Implementing avatar deletion.

![CleanShot 2024-11-26 at 11 40 00](https://github.com/user-attachments/assets/b4101fa3-a2c3-4bcc-8b7d-2860b738ce65)

### Testing Steps

- Run the demo app.
- Go to `Quick Editor`.
- Open the QE.
- Tap on the action button (elipsis) on any avatar).
- Tap on Delete action
  - A confirmation sheet should appear. 
  - The avatar should be deleted after confirmation.
- For 404 errors, the UX will appear as a success.
- For any other error, a toast will appear with a message and the avatar will animate back into the grid.
